### ISSUE TITLE
How to fine-tune for high resolution?

### ISSUE THREAD

**Issue by ilyassmoummad** [RobinWu218/SimDINO#53]
Hi,

Excellent work! We have been using SimDINOv2 for fine-grained representation learning on plant images, and it works really well. We recently applied it in our work: “Fine-grained representation learning of plant images” ( https://arxiv.org/abs/2604.27538).

After pretraining at 224×224 with a per-GPU batch size of 128, I would like to fine-tune the model at higher resolutions (e.g. 512×512). However, due to memory constraints, the maximum batch size I can fit is 32 per GPU, and I have not been able to obtain improvements during this high-resolution adaptation stage.

I tried several modifications, including:

-enabling/disabling reduce_cov
-lowering/increasing the coding rate epsilon
-increasing or fixing the teacher momentum
-lowering the learning rate and/or weight decay

but I still cannot find a stable setup that improves performance with batch size 32, even after 1–10 fine-tuning epochs. Do you have any recommendations or best practices for this kind of high-resolution fine-tuning stage, similar to what is done in DINOv2/DINOv3?

Thank you very much in advance!
